src/main/java/org/fundacionjala/coding/danielM/bank_ocr/BankOcr.java

```diff
@@ -171,4 +175,44 @@ public static String accountRepresentation(final String[] scannedImage) {
 
         return acctRepresentation.toString();
     }
+
+    /**
+     * This method take and scanned account of exactly 9 digits and return
+     * an array representation of them.
+     *
+     * @param scannedAccount String, represents the account.
+     * @return an array of nine string digits.
+     */
+    public static String[] parseScannedNumbers(final String scannedAccount) {
+
+        String[] scannedDigits = {"", "", "", "", "", "", "", "", ""};
```

Comment:
Why do you need to use an array here and have empty strings?
I think you can use a List here